### PR TITLE
Add failure domain labels to created PVs

### PR DESCRIPTION
**Pull request by prydie**
Fixes: #100

### CODE REVIEW

File: pkg/oci/client/client.go [hunk 1]
```diff
@@ -90,6 +91,10 @@ func (p *provisionerClient) TenancyOCID() string {
 	return p.cfg.Auth.TenancyOCID
 }
 
+func (p *provisionerClient) Region() string {
```
Comment (jhorwit2) [inline pkg/oci/client/client.go]: This doesn't match the CCM. It uses `phx` for region and if i'm not mistaken this uses the api region, which is `us-phoenix-1`

Comment (prydie) [reply pkg/oci/client/client.go]: Ah yes, we use `instance.Region` in the CCM:

> The region that contains the Availability Domain the instance is running in.
> Example: phx

File: pkg/oci/client/client.go [hunk 2]
```diff
@@ -102,6 +103,10 @@ func (p *provisionerClient) TenancyOCID() string {
...
```
Comment (prydie) [inline pkg/oci/client/client.go]: Redundant now when using metadata.

File: pkg/provisioner/block/block.go [hunk 3]
```diff
@@ -114,9 +114,13 @@ func (block *blockProvisioner) Provision(options controller.VolumeOptions, ad *i
 	//volumeName := mapVolumeIDToName(*newVolume.Id)
 	filesystemType := resolveFSType(options)
 
-	metadata, err := block.metadata.Get()
-	if err != nil {
-		return nil, err
+	region, ok := os.LookupEnv("OCI_VOLUME_PROVISIONER_REGION")
```
Comment (jhorwit2) [inline pkg/provisioner/block/block.go]: is there a more common env var that should be used? For example, when you look at AWS you always use AWS_REGION?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,16 @@
 # CHANGELOG
 
+## 0.7.0
+
+- Add failure domain labels to created PVs [#101]
```
Comment (prydie) [inline CHANGELOG.md]: Again see comment about not updating CHANGELOG

File: Makefile [hunk 5]
```diff
@@ -116,6 +116,6 @@ version:
 
 .PHONY: run-dev
 run-dev: build
-	NODE_NAME=localhost ./dist/oci-volume-provisioner \
+	NODE_NAME=localhost OCI_VOLUME_PROVISIONER_REGION=iad ./dist/oci-volume-provisioner \
```
Comment (prydie) [inline Makefile]: Perhaps `OCI_VOLUME_PROVISIONER_REGION ?= iad` at the top level would be better to enable overriding.

File: pkg/provisioner/block/block_test.go [hunk 6]
```diff
@@ -97,6 +98,10 @@ func (p *mockProvisionerClient) TenancyOCID() string {
 	return "ocid1.tenancy.oc1..aaaaaaaatyn7scrtwtqedvgrxgr2xunzeo6uanvyhzxqblctwkrpisvke4kq"
 }
 
+func (p *mockProvisionerClient) Region() string {
```
Comment (prydie) [inline pkg/provisioner/block/block_test.go]: Redundant.

File: pkg/provisioner/block/block.go (hunk 3 above)
Comment (prydie) [reply pkg/provisioner/block/block.go]: Good idea. `OCI_REGION` then (though I'd expect that to be the long region name generally...)?

Comment (harveylowndes) [reply pkg/provisioner/block/block.go]: @prydie my thoughts exactly. Seems there needs to be a better distinction between the canonical region and the region (short) across projects.

File: pkg/provisioner/block/block.go [hunk 7]
```diff
@@ -113,13 +114,25 @@ func (block *blockProvisioner) Provision(options controller.VolumeOptions,
 	//volumeName := mapVolumeIDToName(*newVolume.Id)
 	filesystemType := resolveFSType(options)
 
+	region, ok := os.LookupEnv("OCI_REGION")
```
Comment (prydie) [inline pkg/provisioner/block/block.go]: `OCI_SHORT_REGION` I think

**Pull request merged**